extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -728,7 +728,8 @@ Some interactions may result in a transaction. If so, the interactionId in the p
 
 ### Adyen payment refusals
 
-If you receive an HTTP 200 response with an Error or Refused resultCode, check the refusal reason and, if possible, modify your request.
+If you receive an `HTTP 200 response` with an `Error` or `Refused` resultCode from Adyen, a transaction will be added to the commercetools payment object as the refusal result of the payment request.
+Use the commercetools payment `interface interactions` to [troubleshoot](#extension-module-errors) the response.
```

Comment:
Maybe better to link interactions with the commercetools docs? 
Linking to troubleshoot couple lines above does not seem helpful as it does not provide more infos.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/646/commits/8774071da255b4dfc47f8a77d03b5381e20514fc